compiler/rustc_session/src/options.rs

```diff
@@ -252,7 +252,7 @@ macro_rules! options {
         pub const parse_passes: &str = "a space-separated list of passes, or `all`";
         pub const parse_panic_strategy: &str = "either `unwind` or `abort`";
         pub const parse_relro_level: &str = "one of: `full`, `partial`, or `off`";
-        pub const parse_sanitizers: &str = "comma separated list of sanitizers: `address`, `leak`, `memory` or `thread`";
+        pub const parse_sanitizers: &str = "comma separated list of sanitizers: `address`, `leak`, `memory`, `thread` or `hwaddress`";
```

Comment:
Use alphabetic ordering for user facing messages.

Reply:
Done